packages/core/src/WebPlayer.ts

```diff
@@ -73,7 +73,7 @@ export default class WebPlayer extends EventEmitter {
     return this.tech?.isMuted ?? false;
   }
 
-  get isLive() : boolean {
+  get isLive(): boolean {
```

Comment:
To reduce risks for merge conflicts, it is good practice to try to keep only code changes relevant to this feat/issue in the PR.

Reply:
I must've accidentally formatted  the file at some point.. I understand, I will keep it in mind.